Modified nuspec files

daniel-luberda opened the pull request:
Any suggestions?

molinch commented:
I read the changes but could you explain these a bit?

molinch commented:
If you could consider having separate package for FFImageLoading (classic iOS/Android) and WP.
Then Forms should of course contain all 3 platforms.

That would be awesome and we could then roll new packages :+1:

daniel-luberda commented:
I'll do a new commits soon with a separated Windows packages. 

Some of the changes:
- updated projects description
- updated version to 2.0
- changed PCL profile to 259
- added Windows
